woocommerce/woocommerce-android
# [TTP] Improve TTP pin required error handling

Closes: #9221


### Description

Adds specific text and action button that leads to the HW reader purchase page when PIN is required during TTP payment

### Testing instructions

* Try to collect the 1.02 payment with TTP
* Notice the new screen
* Click on the "purchase a reader" button
* Notice a web page

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/bfe1b8d0-1f95-45c2-b573-db8c4b07591e
